### DISCUSSION TITLE
How to change the colors used for the menu?

### DISCUSSION THREAD

**Discussion by shanti2530** [sibiraj-s/ngx-editor#379] [Q&A] [answered]
I would like to specify a color theme for the menu so that it matches the application colors.

**Comment by sibiraj-s** [comment]
You can use the `colorPresets` option in the menu component.

See https://github.com/sibiraj-s/ngx-editor/blob/9d4c9ab34cda070d7d9ac2b9aea73271046f2c2e/projects/ngx-editor/src/lib/modules/menu/menu.component.ts#L57

See docs. https://sibiraj-s.github.io/ngx-editor/#/menu?id=component-props

**Reply by shanti2530** [reply]
@sibiraj-s  I actually want to change the colors of the menu itself and not the allowed text colors.

I was wondering, if there is a way to configure the colors of the menu rather than overriding the css classes directly.

Here's an example of the menu with different colors.
<img width="504" alt="Screenshot 2022-01-29 at 13 51 42" src="https://user-images.githubusercontent.com/1586946/151661844-fcf4ee9e-8151-4939-ad4f-e3e35833bc66.png">

**Reply by sibiraj-s** [reply]
oh. that. I didn't see the keyword theme there 😬 . Yeah. For now you will have to override via css only. may be later I can do something with css variables. 🤔

**Reply by shanti2530** [reply]
no worries. Thanks anyway 🙂

**Reply by sibiraj-s** [reply, marked as answer]
Styling via css variables are now supported in v17.3.0

[a comment by an automated account is not shown]
